src/main/java/com/hcl/appscan/jenkins/plugin/scanners/StaticAnalyzer.java

```diff
@@ -72,5 +78,13 @@ public static final class DescriptorImpl extends ScanDescriptor {
 		public String getDisplayName() {
 			return STATIC_ANALYZER;
 		}
+
+		public FormValidation doCheckOpenSourceOnly(@QueryParameter Boolean openSourceOnly, @RelativePath("..") @QueryParameter String credentials, @AncestorInPath ItemGroup<?> context) {
```

Comment:
Instead of having to show an error if the user mistakenly selects this option, can we just disable it if AppScan 360 credentials are selected?